data/sql_updates/candidate_history.sql

```diff
@@ -66,7 +66,8 @@ left join disclosure.cand_inactive inactive on
     fec_yr.cand_id = inactive.cand_id and
     fec_yr.fec_election_yr < inactive.election_yr
 left join staging.ref_pty ref_party on fec_yr.cand_pty_affiliation = ref_party.pty_cd
-where max_cycle >= :START_YEAR
+where max_cycle >= :START_YEAR and
+    fec_yr.cand_id not in (select distinct cmte_id from unverified_filers_vw where cmte_id similar to '(P|S|H)%')
```

Comment:
I did a double-take when I first saw this because I was like "d'oh, how could I miss that..." - turns out that view only has one ID column, `cmte_id`.  Counter-intuitive, I know!

There still appears to be an issue with these though, and I need to fix the test database again.  Might be worth pairing for a few minutes on this because by all accounts this SQL should do the trick, but with it in place no records end up appearing in these materialized views.